example/cmd/device-simple/Attribution.txt

```diff
@@ -96,6 +96,9 @@ https://github.com/pmezard/go-difflib/blob/master/LICENSE
 stretchr/testify (MIT) https://github.com/stretchr/testify
 https://github.com/stretchr/testify/blob/master/LICENSE
 
+ugorji/go (MIT) https://github.com/ugorji/go
+https://github.com/ugorji/go/blob/master/LICENSE
+
```

Comment:
Please update the reference to the CBOR library per https://github.com/edgexfoundry/edgex-go/pull/2490